src/ViewInstance.js

```diff
@@ -561,31 +561,27 @@ class ViewInstance extends React.Component {
                 disabled={!marcRecord}
               />
             )}
-            {!openedFromBrowse && (
```

Comment:
I think we can remove all uses of `openedFromBrowse` - one more place is in `showInventoryMenuSection` variable and there rest